spec/index.html

```diff
@@ -310,7 +310,9 @@ <h2>Terminology</h2>
       with <code>_:</code> that is used as an identifier for an
       <a>blank node</a>. <a>Blank node identifiers</a>
       are typically implementation-specific local identifiers; this document
-      specifies an algorithm for deterministically specifying them.</dd>
+      specifies an algorithm for deterministically specifying them.
+      A blank node not given an explicit identier is called an "unlabeled blank node."
+      Additionally, blank nodes may be explicitly labeled with an identifier.</dd>
```

Comment:
```suggestion
      Additionally, blank nodes may be explicitly labeled with an identifier.</dd>
```
"Blank nodes may be made unblank, though they remain blank."  There's gonna be a LOT of confusion coming from this.

Reply:
Need more work on the "label" vs "identifier" bit. The spec originally used "labeled", but from RDF Concepts its "Blank Node identifier". In some cases, the word "label" works better, but this still isn't quite right.

In the abstract syntax, blank nodes do not have identifiers, but when converted to N-Quads, they are (of course) given identifiers. And typically, if they were assigned identifiers in a serialization, that identifier is re-used when re-serializing, which is why they show up here.

Reply:
It's probably worth using the "official" terminology, and including *at least* a link to the [relevant section of RDF Concepts](https://www.w3.org/TR/rdf11-concepts/#section-blank-nodes), and possibly a paraphrasing of the note included there ... the key detail being that these identifiers/labels are intended to be ephemeral, subject to the specific serialization and/or other processing in which they're minted, and cannot be expected to be used beyond such processing and/or serialization.

Reply:
Sure, except that they _are_ significant in the resulting normalized dataset.

Reply:
Yes, they are significant in the resulting normalized dataset serialization, but they may (and probably will) not persist when that dataset is loaded into a data store or otherwise transformed from the normalized dataset.

Reply:
Probably need to update to eliminate any dependency on the bn identifier from the input, and just talk about Blank Nodes. Identifiers become important when they are candidate or final canonical identifiers.

Reply:
I'd say a separate PR that focuses on terminology (identifier here / label there, whatever) would be in order. I'd rather not hold up this PR on account of finding the best way to clarity on this issue.

Reply:
I'm going to defer any further updates on how we talk about blank nodes to #46.